tlvc/arch/i86/drivers/block/directhd.c

```diff
@@ -565,7 +560,7 @@ void directhd_release(struct inode *inode, struct file *filp)
 
     access_count[target]--;
```

Comment:
Always decrementing the driver reference counter here but only incrementing it for block device opens will cause problems. Why is that being done, just to skip the invalidate routines on close?

Reply:
Thanks for the code review, @ghaerr.

Actually, the block device close routine is never called from the raw driver. Not obvious since open is called. Then again there is nothing to do when closing a raw access.

-M